python/cuml/test/test_dbscan.py

```diff
@@ -84,7 +82,7 @@ def test_dbscan(datatype, input_type, use_handle,
                          stress_param(500000)])
 def test_dbscan_sklearn_comparison(name, nrows):
     default_base = {'quantile': .3,
-                    'eps': .3,
+                    'eps': .5,
```

Comment:
Did sklearn vs dbscan not get the correct results without this change?

Reply:
No, i just thought that i should keep it consistent with the above function. I can change it back if you prefer.

Reply:
No change necessary. Glad the results match!